docs/antrea-ipam.md

```diff
@@ -278,8 +278,8 @@ where the underlay router will route the traffic to the destination VLAN.
 ### Requirements for this Feature
 
 As of now, this feature is supported on Linux Nodes, with IPv4, `system` OVS datapath
-type, `noEncap`, `noSNAT` traffic mode, and `AntreaProxy` feature enabled. Configuration
-with `ProxyAll` feature enabled is not verified.
+type, `noEncap`, `noSNAT` traffic mode, and Antrea Proxy enabled. Configuration
+with `proxyAll` enabled is not verified.
```

Comment:
Worthy to check all `ProxyAll` and update `ProxyAll` to `proxyAll` if necessary.
I can see there are inconsistent in some docs.

Reply:
Will check.

Reply:
I found that every `ProxyAll (Without quotting)` is at the beginning of sentences. I don't think we need to update them.

Reply:
Only when it refers to the `proxyAll` field in the YAML, we should we use `proxyAll`.

Reply:
> Only when it refers to the proxyAll field in the YAML, we should we use proxyAll.

I may not get what you mean. Do you mean that we should use `proxyAll` in the YAML comments, even if the word is at the beginning of sentences?

Reply:
'proxyAll` here should refer to the `proxyAll` field in `antrea-agent.conf` right? In this case, we should use `proxyAll` as Lan suggested, not `ProxyAll`, as the field is named `proxyAll`. It does not matter whether it begins a sentence or not.

Reply:
Thanks, got that. If so, how about this: `Option proxyAll`?

Reply:
I think the current version here "Configuration
with `proxyAll` enabled is not verified" is fine.

Reply:
> Thanks, got that. If so, how about this: Option proxyAll?

I meant that how about changing the `ProxyAll tells antrea-agent .....` at the beginning of sentens in YAML files to `Option proxyAll tells antrea-agent`.

> I think the current version here "Configuration
with proxyAll enabled is not verified" is fine.

I didn't change this one. @jianjuns

Reply:
OK. I think we can say "proxyAll tells..."